process explorer/backend/ProcessExplorer/Processes/Interfaces/IProcessMonitor.cs

```diff
@@ -15,6 +15,7 @@ public interface IProcessMonitor
         void KillProcessByName(string processName);
         void SetSubribeUrl(string url);
         void SetComposePID(int pid);
+        void SetDelay(int delay);
```

Comment:
I think the word "delay" alone does not communicate what is being delayed. We need something like "SetDeadProcessRemovalDelay".